background_jobs/config.example.yaml

```diff
@@ -0,0 +1,9 @@
+CleanBlocks:
+  ExecutionIntervalInHours: 24 # how often to run the job
+  BlocksRetentionDays: 14 # delete records older thhan
+  Host:  "localhost" # db connection host
+  Port:  5432  # db connection port
+  Username: "postgres" # db connection username
+  Password: "postgres" # db connection password
+  DBName:  "postgres"  # db connection database
```

Comment:
DBName is `blocktx`

Reply:
this is `example` configuration that can be different per setup. See the Readme ;)